src/controllers/places/add_place.js

```diff
@@ -25,23 +25,25 @@ module.exports = (req, res) => {
   getLatLng(req.body.address, (err, response) => {
     if (!err) {
       apiBody.location = [response.lat, response.lng];
```

Comment:
These need to be the other way round: https://docs.mongodb.com/manual/geospatial-queries/#legacy-coordinate-pairs